### PR TITLE
Fix reflow zooming bug for OData libraries and reference tables

### PR DESCRIPTION
**Pull request by wandeg**
Before
before-zoom

After
fix-overflow

### CODE REVIEW

**Review by marabooy — APPROVED** [review]

File: _includes/scripts.html [hunk 1]
```diff
@@ -78,4 +78,37 @@
...
+        isReflowed = true;
+      }
+
+      // Zoom < 300%
+      if (isReflowed && window.innerWidth > 640){
+        isReflowed = false;
+        $("b.cell-label").remove();
+        $("span.cell-content").contents().unwrap();
+      }
+
```
Comment (marabooy) [inline _includes/scripts.html:112]:
```suggestion
```

File: public/css/site.css [hunk 2]
```diff
@@ -194,4 +194,81 @@ code {
...
+        float:right;
+      }
+
+  table.reflows tr:nth-child(even) {
+        background-color: rgba(255, 255, 255, 0.25);
+  }
+  table.reflows tr:nth-child(odd) {
+        background-color: #F9F9F9;
+  }
+
```
Comment (marabooy) [inline public/css/site.css:269]:
```suggestion
```

File: public/css/site.css [hunk 3]
```diff
@@ -194,4 +194,81 @@ code {
     .tutorial-sidebar .nav>.active>ul {
     display: block;
 	}
-}
\ No newline at end of file
+}
+
+
```
Comment (marabooy) [inline public/css/site.css:199]:
```suggestion
```

**Review by marabooy — APPROVED** [review]

File: _includes/scripts.html [hunk 4]
```diff
@@ -78,4 +78,36 @@
...
+            // add titles to cells
+            $('table.reflows')
+              .find('tr').find('td:eq('+index+')').wrapInner('<span class="cell-content"></span>').prepend( title );
+        });
+        isReflowed = true;
+      }
+
+      // Zoom < 300%
+      if (isReflowed && window.innerWidth > 640){
```
Comment (gathogojr) [inline _includes/scripts.html:107]: @wandeg Should you have an `else if` here or do you actually need the conditional expression on this `if` block here and the other on the `if` block on Line 91 both be evaluated always?

File: public/css/site.css [hunk 5]
```diff
@@ -194,4 +194,79 @@ code {
     .tutorial-sidebar .nav>.active>ul {
     display: block;
 	}
-}
\ No newline at end of file
+}
```
Comment (gathogojr) [inline public/css/site.css:197]: @wandeg The alignment/indentation on the CSS introduced looks wonky

**Review by habbes — APPROVED** [review]

**Review by gathogojr — APPROVED** [review]

**Pull request merged**